ubuntu_install_requirements.sh

```diff
@@ -69,6 +69,7 @@ EOF
   sudo mkdir -p /etc/docker
   sudo mv daemon.json /etc/docker/daemon.json
   sudo systemctl restart docker
+  sudo usermod -aG docker "${USER}"
```

Comment:
there is also a docker install in the centos requirements. Can you add it there too ?

Reply:
but we don't use docker in Centos, why would we need it?

Reply:
Ok. Working on it.

Reply:
Done